### ISSUE TITLE
Turbo stream - not initialising

### ISSUE THREAD

**Issue by phoozle** [SimoTod/alpine-turbo-drive-adapter#32]
On a fresh page load (no Turbo drive) my Alpine components initialise fine when loaded in via a turbo-stream.

However if I navigate away to a different page via Turbo Drive and revisit the original page, the same Alpine components delivered via the turbo-stream no longer get initialised.

I have in the meantime come up with the following workaround using a Stimulus controller:
```
// alpine_init_controller.js
import { Controller } from "stimulus"

export default class extends Controller {
  connect() {
    Alpine.initializeComponent(this.element)
  }
}
```

```
<turbo-stream action="update" target="my_target">
  <template>
    <div x-data="{}" data-controller="alpine-init">
     ...
    </div>
  </template>
</turbo-stream>
```

**Comment by SimoTod** [comment]
Thanks @phoozle 
The current version is a porting from the old turbolinks so turbo-stream, being a new feature, is not supported yet.
I'm happy to accept a PR if you are willing to contribute otherwise I can look into it next week.

**Comment by SimoTod** [comment]
Hi @phoozle 
I tried a simple example with a form and an HTTP response returning a turbo stream snippet containing an Alpine component and it seems to work correctly. Can you give me an example where it does not work? Thanks

**Comment by SimoTod** [comment]
@phoozle would you mind testing https://cdn.jsdelivr.net/gh/SimoTod/alpine-turbo-drive-adapter@bug/turbo-stream/dist/alpine-turbo-drive-adapter.min.js and let me know if it fixes the bug you reported? Thanks

**Comment by SimoTod** [comment]
Fixed in v1.1.0

**Comment by phoozle** [comment]
Sorry @SimoTod somehow missed your message. This solved my issues! You're a legend
